Bench.php

```diff
@@ -311,6 +315,43 @@ public function getLongest()
     }
 
     /**
+     * Pause all Marks.
+     *
+     * @return  array
+     */
+    public function pause()
+    {
+        $runningMarks = [];
+
+        foreach ($this as $mark) {
+            if ($mark->isRunning()) {
```

Comment:
The result with the Pause test is:
```
Ggestleplusbeau__global__  ||||||||||||||||||||||||||||||||||||||||||||||||||||     1ms, 100.0%
one         ||||                                                     0ms,   7.8%
oneBis      ||||                                                     0ms,   7.4%
two         |                                                        0ms,   2.1%
three       |                                                        0ms,   2.4%
```
Match the new values of oneBis with the previous ones.